src/hooks/common.ts

```diff
@@ -6,7 +6,7 @@ export const resolveDeviceType = async (
 	api: sbvrUtils.PinejsClient,
 	request: sbvrUtils.HookRequest,
 	fkValue: string,
-): Promise<deviceTypes.DeviceType> => {
+) => {
```

Comment:
It's good to have explicit types on function definitions.

```suggestion
): Promise<void> => {
```